### DISCUSSION TITLE
No puedo acceder al Link del formulario para entregar Katas

### DISCUSSION THREAD

**Discussion by DiegoGarciaG00** [LaunchX-InnovaccionVirtual/CursoIntroPython#246] [Q&A] [answered]
Alguien ya pudo entregar las Katas en el link que se puso?
cuando intento acceder me dice que no tengo permiso para ese formulario, pero si estoy entrando desde la cuenta de Innovación
Saben si ya los habilitaron
![Captura de Pantalla 2022-02-14 a la(s) 16 23 06](https://user-images.githubusercontent.com/52545906/153956550-11784655-fc9d-4083-8faf-f2265de1640d.png)
?

**Comment by JesusHerreraB** [comment]
Me paso lo mismo, solo cambié el correo con el que estaba accediendo! Usa el launch@innovaccion.mx

**Comment by doguedogue** [comment, marked as answer]
Comentó la sherpa que abrieras en una ventana de incognito y usaras tus credenciales de @innovaccion
